backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc

```diff
@@ -23,7 +23,7 @@ You create a `Secret` for the backup location and then you install the Data Prot
 
```

Comment:
You can probably remove the [link at line 16](https://github.com/openshift/openshift-docs/pull/64209/files#diff-5b787fc42ecab6023d2af95651a5e185c4538a27237c73d8ef9b8224ec73fdafR16), as it seems to link to the assembly itself.